docs/lakehouse-iceberg-catalog.md

```diff
@@ -111,10 +111,33 @@ For `bucket` and `truncate`, the first argument must be integer literal, and the
 
 ### Table distributions
 
-- Doesn't support `Distribution`, you should use `BucketPartition` instead.
+- Gravitino used by default `NoneDistribution`.
+```json
+{
+  "strategy": "hash"
+}
```

Comment:
You should also change this json string as other fields cannot be empty.

Besides, you'd better use the codetab plugin to support different languages, please refer to here https://github.com/datastrato/gravitino/blob/8b753fd9a00b3cbb84dbaf86996f744bb7d71fe7/docs/manage-metadata-using-gravitino.md?plain=1#L33